### DISCUSSION TITLE
How can I save the output for FindConservedMarkers?

### DISCUSSION THREAD

**Discussion by pcitrano3** [satijalab/seurat#6159] [Q&A] [answered]
Hello. This might be a really simple question, but I am currently using the FindConservedMarkers() function for my integrated dataset in Seurat, and I am wondering if I can save the output table that lists the genes and their p-values etc. into an Excel workbook or somewhere in my computer like my documents. I want to be able to pull up this information in the future without having to re-run the entire code all over. 

I am new to R and Seurat, so any help with saving my work would be greatly appreciated. I have tried looking online but I can not figure it out.

**Comment by samuel-marsh** [comment, marked as answer]
Hi,

You can save the output as csv file by using:

```
markers <- FindAllMarkers(…)

write.csv(markers, “path/to/save/filename.csv”)
```

You can also save as R object that can easily be reloaded into R quickly
```
saveRDS(markers, “path/to/save/filename.rds”)

# load into R
markers <- readRDS(“path/to/save/filename.rds”)
```

Best,
Sam
